Tests/BackendTest/FindCoursesTest.cs

```diff
@@ -0,0 +1,113 @@
+using FluentAssertions;
+using FRITeam.Swapify.APIWrapper;
+using FRITeam.Swapify.Backend;
+using FRITeam.Swapify.Entities;
+using Microsoft.Extensions.Logging;
+using MongoDB.Driver;
+using Moq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BackendTest
+{
+    [Collection("Database collection")]
+    public class FindCoursesTest : IClassFixture<Mongo2GoFixture>
+    {
+        private readonly Mongo2GoFixture _mongoFixture;
+        private readonly IMongoDatabase _database;
+        private readonly Mock<ILogger<CourseService>> _loggerMockCourse;
+
+        public FindCoursesTest(Mongo2GoFixture mongoFixture)
+        {
+            _mongoFixture = mongoFixture;
+            _loggerMockCourse = new Mock<ILogger<CourseService>>();
+            _database = _mongoFixture.MongoClient.GetDatabase("StudentsDB");        
+        }
+
+        [Fact]
+        public async Task FindCoursetWithoutWritingAccents()
+        {
+            var schoolScheduleProxy = new SchoolScheduleProxy();
```

Comment:
Maybe make this local variables as class attributes will be better if all tests are run at the same time.